src/io/flutter/FlutterBundle.properties

```diff
@@ -206,7 +206,6 @@ info.level.title=Info
 warning.level.title=Warning
 severe.level.title=Severe
 shout.level.title=Shout
-# End Flutter Log Color Page
```

Comment:
was deleting this line intentional?

Reply:
Yes - there's a comment a few lines above mentioning the start of this section; I don't think we generally need to bookmark sections in this file (we don't other places).